package.json

```diff
@@ -232,5 +232,13 @@
     "safari5": [
       "Safari < 5"
     ]
+  },
+  "dependencies": {
+    "opencollective-postinstall": "^2.0.2",
+    "opencollective": "^1.0.3"
```

Comment:
This package is not required. `opencollective-postinstall` is alternative and lite version of `opencollective` cli.
also , I think ,`opencollective` still has some bugs which are not resolved and thats why `opencollective-postinstall`  was made to address that.

Reply:
Exactly, `opencollective-cli` is not actively maintained anymore, `opencollectiv-postinstall` serves the purpose here.